src/rise-twitter.js

```diff
@@ -113,9 +113,9 @@ export default class RiseTwitter extends HTMLElement {
     }
   }
 
-  _playPreview() {
+  _playInPreview() {
     this.tweet.displayFillerTweets();
-    this._play();
```

Comment:
Fixes typeerror we're seeing in preview. We shouldn't use `play()` function since it's for non-preview.